src/utilities/bcl/RemoteBCL.cpp

```diff
@@ -98,7 +112,8 @@ RemoteBCL::RemoteBCL()
     m_lastTotalResults(0),
     m_apiVersion("2.0"),
     validProdAuthKey(false),
-    validDevAuthKey(false) {
+    validDevAuthKey(false),
+    m_timeOutSeconds(120) {
```

Comment:
Actual fix for #4014: currently downloading Components is super slow, takes 66 seconds on my machine. Downloading Measures on the other hand is much faster...